configure.ac

```diff
@@ -5998,6 +6001,48 @@ then
     AM_CFLAGS="$AM_CFLAGS -DHAVE_DH_DEFAULT_PARAMS"
 fi
 
+
+AC_ARG_WITH([max-rsa-bits],
+    [AS_HELP_STRING([--with-max-rsa-bits=number],[number of bits to support for RSA, DH, and DSA keys])],
+    [WITH_MAX_ASSYM_KEY_BITS=$withval],
+    [WITH_MAX_ASSYM_KEY_BITS="$DEFAULT_MAX_ASSYM_KEY_BITS"])
+
+if test -n "$WITH_MAX_ASSYM_KEY_BITS"; then
```

Comment:
WITH_MAX_ASSYM_KEY_BITS -> WITH_MAX_ASYM_KEY_BITS

Reply:
oh for crying out loud, I've been misspelling that word for decades, on and off.

maybe this will help me:

```
From WordNet (r) 3.0 (2006) [wn]:

  asymmetric
      adj 1: characterized by asymmetry in the spatial arrangement or
             placement of parts or components [syn: {asymmetrical},
             {asymmetric}] [ant: {symmetric}, {symmetrical}]
```

thanks!